internify/src/components/molecules/Score.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { Chip } from "@material-ui/core";
+// import blue from '@material-ui/core/colors/blue';
+
+const Score = ({score}) => {
+    return (
+        <div className="score_container">
+            <h3>Score</h3>
+
+            {(score >= 0.85) && <Chip 
+                size="small"
+                label="Excellent"
+                />}
+
+            {(score >= 0.70 && score <= 0.84) && <Chip
+                size="small"
+                label="Good"
+                color="primary" />}
+
+            {(score >= 0.50 && score <= 0.69) && <Chip 
+                size="small"
+                label="Satisfactory" />}
+
+            {(score < 0.50) && <Chip 
+                size="small"
+                label="Poor" />}
```

Comment:
It's great you guys have decided on the thresholds (makes your lives easier), and I like these thresholds. I would probably change this label from "Poor" to something a little more positive. Can't think of examples right now, but in industry we'd normally want to avoid negative words.